scene/gui/label.cpp

```diff
@@ -454,7 +454,7 @@ void Label::regenerate_word_cache() {
 			total_char_cache++;
 		}
 
-		if ((autowrap && (line_width >= width) && ((last && last->char_pos >= 0) || separatable)) || insert_newline) {
+		if ((autowrap && (line_width >= width) && last && ((last->char_pos >= 0) || separatable)) || insert_newline) {
```

Comment:
This seems like it could have a different result with this change. Have you verified that this won't change the functionality?

Reply:
A test:
```
int main()
{
    for(int a=0;a<2;a++)
	for(int b=0;b<2;b++)
	for(int c=0;c<2;c++)
	for(int d=0;d<2;d++)
	for(int e=0;e<2;e++)
	for(int f=0;f<2;f++)
	if( ((a && b && ((c && d) || e)) || f) != ((a && b && c && ((d) || e)) || f) )
        cout<<a<<","<<b<<","<<c<<","<<d<<","<<e<<","<<f<<","<<"\n";
    return 0;
}
/*
//Output or non-matching cases
1,1,0,0,1,0,
1,1,0,1,1,0,
*/
```
where a,b,c,d,e,f are the alias in same order.
@LikeLakers2 is right.